### PR TITLE
Handle new configs for 7.12 / 8.0.0

### PR DESCRIPTION
**Pull request by andrewvc**
Handles the new config style delivered in https://github.com/elastic/beats/pull/23467/files#diff-7f1e5387c4757cca1e98483a5678e377a28ca5f9d77b267a4121a14249c96b82R7 which are required prior to the 7.12.0 release.

WIP

### CODE REVIEW

**Comment by andrewvc** [comment]
@kuisathaverat thoughts on the cryptic linting error on the build here?

```
[2021-02-23T23:20:33.920Z] Recording test results

[2021-02-23T23:20:34.291Z] [Checks API] No suitable checks publisher found.
```

File: examples/docker/heartbeat-8.0.0-docker.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        step("hover over products menu", async () => {
+            await page.hover('css=[data-nav-item=products]');
+        });
+- type: browser
+  id: todos
+  name: Todos
+  schedule: "@every 1m"
+  source:
+    local:
+      path: "/opt/elastic-synthetics/examples/todos"
```
Comment (kuisathaverat) [inline examples/docker/heartbeat-8.0.0-docker.yml:22]:
```suggestion
      path: "/opt/elastic-synthetics/examples/todos"

```

Comment (kuisathaverat) [reply examples/docker/heartbeat-8.0.0-docker.yml:22]: it needs a new line at the end of the file.

**Review by vigneshshanmugam — APPROVED** [review]
LGTM, Let's merge this @andrewvc I see no harm in doing this. We can clean it up once we release 7.12.

**Comment by vigneshshanmugam** [comment]
Closing this PR in favor of #257

**Pull request closed without merging**